src/chat.rs

```diff
@@ -3470,7 +3481,7 @@ pub async fn create_group_chat(
 
     let chat_id = ChatId::new(u32::try_from(row_id)?);
     if !is_contact_in_chat(context, chat_id, ContactId::SELF).await? {
```

Comment:
Btw, what is the purpose of this check? Does the result depend on `protect` somehow?